tests/wallet.rs

```diff
@@ -336,7 +336,7 @@ mod node {
             .into_iter()
             .map(|value| {
                 let obfuscated = (rng.next_u32() & 1) == 1;
-                let idx = rng.next_u64() % MAX_KEY as u64;
+                let idx = rng.next_u64() % (MAX_KEY) as u64;
```

Comment:
You don't need to wrap the `MAX_KEY` in parentheses, the previous code would works fine.